backend/src/bundles/chat-messages/chat-message.model.ts

```diff
@@ -0,0 +1,66 @@
+import { type JSONSchema, Model, type RelationMappings } from 'objection';
+
+import {
+    AbstractModel,
+    ChatMessagesTableColumn,
+    DatabaseTableName,
+    UserDetailsTableColumn,
+} from '~/common/packages/database/database.js';
+
+import { UserDetailsModel } from '../user-details/user-details.model.js';
+
+class ChatMessageModel extends AbstractModel {
+    public 'senderId': string;
+    public 'receiverId': string;
+    public 'chatId': string;
+    public 'message': string;
+    public 'isRead': boolean;
+    public 'sender': UserDetailsModel;
```

Comment:
why we need sender and senderId?

Reply:
I'm getting this from relation, and I need it to prevent errors with types.
Also, in recent changes, I've added additional fields there